hbase-replication/src/main/java/org/apache/hadoop/hbase/replication/ReplicationQueueStorage.java

```diff
@@ -184,4 +185,13 @@ void removeLastSequenceIds(String peerId, List<String> encodedRegionNames)
    * @return Whether the replication queue table exists
    */
   boolean hasData() throws ReplicationException;
+
+  // the below 3 methods are used for migrating
```

Comment:
Minor: would still be nice to have some javadoc for these 3 methods

Reply:
Ack.